packages/core/src/App/Containers/Layout/header/trading-hub-onboarding.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import { Icon, Popover } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+
+const TradingHubOnboarding = observer(() => {
+    const history = useHistory();
+    const { traders_hub, ui } = useStore();
+    const { setIsOnboardingVisited } = traders_hub;
+    const { is_dark_mode_on } = ui;
+
+    return (
+        <div className='trading-hub-header__tradinghub--onboarding'>
+            <div className='trading-hub-header__tradinghub--onboarding--logo'>
+                <Popover
+                    classNameBubble='account-settings-toggle__tooltip'
+                    alignment='bottom'
+                    message={<Localize i18n_default_text='View onboarding' />}
```

Comment:
cant we use localize function?

Reply:
note: reverted to Localize based on latest info from yashim.